proto/bragi/common.proto

```diff
@@ -49,20 +49,28 @@ message ControlErrorAnnouncement {
   optional string message = 2;
 }
 
+message MatchTimeline {
+  repeated Match matches = 1;
+}
+
 message Match {
   string match_urn = 1;
-  string tournament_urn = 2;
+  string tournament_urn = 2 [deprecated=true];
   Sport sport = 3;
-  string home_team_urn = 4;
-  string away_team_urn = 5;
+  string home_team_urn = 4 [deprecated=true];
+  string away_team_urn = 5 [deprecated=true];
   uint32 home_score = 6;
   uint32 away_score = 7;
   MatchStatusType match_status = 8;
   google.protobuf.Timestamp plannet_start = 9;
+  Tournament tournament  = 10;
+  Team home_team = 11;
+  Team away_team = 12;
 }
 
-message MatchTimeline {
-  repeated Match matches = 1;
+message Tournament {
+  string tournament_urn = 1;
```

Comment:
Is there any reason why these fields are prefixed by `tournament_` ? Its already in message name__

Reply:
Changed